codegen/src/main/ops/org/nd4j/codegen/ops/Linalg.kt

```diff
@@ -0,0 +1,200 @@
+package org.nd4j.codegen.ops
+
+import org.nd4j.codegen.api.DataType
+import org.nd4j.codegen.api.Language
+import org.nd4j.codegen.api.doc.DocScope
+import org.nd4j.codegen.dsl.*
+
+fun Linalg() =  Namespace("Linalg") {
+    //val namespaceJavaPackage = "org.nd4j.linalg"
+
+    Op("Cholesky") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.transforms"
+        javaOpClass = "Cholesky"
+        Input(DataType.NUMERIC, "input") { description = "Input tensor with inner-most 2 dimensions forming square matrices" }
+        Output(DataType.NUMERIC, "output"){ description = "Transformed tensor" }
+
+        Doc(Language.ANY, DocScope.ALL){
+            """
+             Computes the Cholesky decomposition of one or more square matrices.
+            """.trimIndent()
+        }
+    }
+
+    Op("Lstsq") {
+        javaPackage = "org.nd4j.linalg.api.ops.custom"
+        javaOpClass = "Lstsq"
+
+        Input(DataType.NUMERIC, "matrix") {description = "input tensor"}
+        Input(DataType.NUMERIC, "rhs") {description = "input tensor"}
+        Arg(DataType.FLOATING_POINT, "l2_reguralizer") {description = "regularizer"}
+        Arg(DataType.BOOL, "fast") {description = "fast mode, defaults to True"; defaultValue = true}
+        Output(DataType.FLOATING_POINT, "output"){ description = "Transformed tensor" }
+
+        Doc(Language.ANY, DocScope.ALL){
+            """
+             Solver for linear squares problems.
+            """.trimIndent()
+        }
+    }
+
+    Op("Solve") {
+        javaPackage = "org.nd4j.linalg.api.ops.custom"
+        javaOpClass = "LinearSolve"
+
+        Input(DataType.NUMERIC, "matrix") {description = "input tensor"}
+        Input(DataType.NUMERIC, "rhs") {description = "input tensor"}
+        Arg(DataType.BOOL, "adjoint") {description = "adjoint mode, defaults to False"; defaultValue = false}
+        Output(DataType.FLOATING_POINT, "output"){ description = "Output tensor" }
+
+        Doc(Language.ANY, DocScope.ALL){
+            """
+             Solver for systems of linear questions.
+            """.trimIndent()
+        }
+    }
+
+    Op("TriangularSolve") {
+        javaPackage = "org.nd4j.linalg.api.ops.custom"
+        javaOpClass = "TriangularSolve"
+
+        Input(DataType.NUMERIC, "matrix") {description = "input tensor"}
+        Input(DataType.NUMERIC, "rhs") {description = "input tensor"}
+        Arg(DataType.BOOL, "lower") {description = "defines whether innermost matrices in matrix are lower or upper triangular"}
+        Arg(DataType.BOOL, "adjoint") {description = "adjoint mode"}
+        Output(DataType.FLOATING_POINT, "otuput")
```

Comment:
typo: "otuput" -> "output"
Same type in a few other places.

Reply:
Fixed.